### ISSUE TITLE
Hiker's feet

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10599]
**Preferred term label:**

**Synonyms**
hyperkeratosis of the feet
hyperkeratotic feet

**Definition (free text, please give PubMed ID)**
PMID: 32284267 “**Hiker's feet**” is a more recently described cutaneous finding, and is essentially mechanic's hands of
the feet, with similar hyperkeratotic eruptions on the toes and feet.

PMID: 35382516 : Mechanic’s hand represents the typical cutaneous manifestation of ASSD. The novel finding of **Hiker’s feet** is the 
equivalent of mechanic’s hands, newly recognized in the recent years. These are characterized by hyperkeratotic scaly fissured plaques on the fingers/palms or soles, generally more evident on the lateral aspect of index fingers. Mechanic’s hand is suggestive of but not specific to ASSD and can be found in IIM associated with other autoantibodies. Presence of mechanic’s hand in ASSD is associated with a higher rate of ILD in some studies.

**Parent term (use hpo.jax.org/app)**
Hyperkeratosis HP:0000962

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:81 Antisynthetase syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001014
- label: Hiker's feet
- parents: [Plantar hyperkeratosis (HP:0007556)](https://hpo.jax.org/app/browse/term/HP:0007556)
- pmid: PMID:32284267|PMID:35382516
- def: Hyperkeratotic scaly fissured plaques on the fingers/palms or soles, generally more evident on the lateral aspect of index fingers.
- comment: Hiker's feet is a finding that is analogous to mechanic's hands with similar hyperkeratotic eruptions on the toes and feet, and is a manifestation of antisynthetase syndrome.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10599
- synonyms:
